__std__/cells/plutus-apps/devshells/plutus-apps-shell.nix

```diff
@@ -153,8 +153,8 @@ inputs.std.lib.dev.mkShell {
 
     # This is used by doc/conf.py to link plutus-core read-the-docs pages
     {
-      name = "PLUTUS_CORE_OBJECTS_INV";
-      value = cell.library.plutus-core-objects-inv;
+      name = "COMBINED_PLUTUS_CORE_HADDOCK";
```

Comment:
This is what we want if we wish to include haddock from the plutus packages. (more comments below)

Reply:
We need to of course be careful that we don't accidentally depend on our _own_ haddock!